corehq/apps/accounting/management/commands/hide_invoices_by_id.py

```diff
@@ -24,7 +24,7 @@ def handle(self, *args, **options):
 
         for invoice_id in args:
             try:
-                invoice = Invoice.objects.get(pk=invoice_id)
+                invoice = super(InvoiceBaseManager, Invoice.objects).get_queryset().get(pk=invoice_id)
```

Comment:
only when you want to override the default filter ```is_hidden_to_ops=False```.  These instances should be rare (for example, now it's only required in one management command and in a test - every other ```Invoice.objects``` call precedes normally)